h2gis-api/src/main/java/org/h2gis/api/ScalarFunction.java

```diff
@@ -31,19 +31,23 @@
  */
 public interface ScalarFunction extends Function {
 
-    /** Boolean, Deterministic functions must always return the same value for the same parameters.
-     *  The result of such functions is cached if possible. */
+    /**
+     * Boolean, Deterministic functions must always return the same value for the same parameters.
+     * The result of such functions is cached if possible.
+     */
     String PROP_DETERMINISTIC = "deterministic";
 
-    /** Boolean, if nobuffer is true then this function will be called more often but will not cache the results in
-     *  memory nor files */
+    /**
+     * Boolean, if nobuffer is true then this function will be called more often but will not cache the results in
+     * memory nor files
+     */
```

Comment:
`NOBUFFER` clause of `CREATE ALIAS` has no effect in recent versions of H2 and it is accepted only for compatibility with older versions.

Reply:
Ok, thanks, I'll remove it.